epub33/core/index.html

```diff
@@ -5031,34 +5024,41 @@ <h5>Other <code>nav</code> Elements</h5>
       …
    &lt;/ol>
 &lt;/nav></pre>
-						</aside>
-					</section>
+					</aside>
 				</section>
+			</section>
 
-				<section id="sec-nav-def-hidden">
-					<h4>The <code>hidden</code> attribute</h4>
-
-					<p>In some cases, <a>EPUB Creators</a> might wish to hide parts of the navigation data within the
-						content flow (i.e., the Reading System's principal rendering of the <a>spine</a> contents). A
-						typical example is the <a href="#sec-nav-pagelist">list of page breaks</a>, which EPUB Creators
-						do not want Reading Systems to render as part of the content flow.</p>
-
-					<p> While the <a href="https://www.w3.org/TR/CSS2/visuren.html#propdef-display"><code>display</code>
-							property</a> [[CSSSnapshot]] controls the visual rendering of EPUB Navigation Documents in
-						Reading Systems with <a>Viewports</a>, not all Reading Systems provide such an interface. To
-						control rendering across all Reading Systems, EPUB Creators MUST use the [[HTML]] <a
-							data-cite="html#the-hidden-attribute"><code>hidden</code></a> attribute to indicate which
-						(if any) portions of the navigation data are excluded from rendering in the content flow. The
-							<code>hidden</code> attribute has no effect on how Reading Systems render the navigation
-						data outside of the content flow (such as in dedicated navigation user interfaces provided by
-						Reading Systems).</p>
-
-					<aside class="example" title="Hiding a nav element in spine">
-						<p>In this example, the presence of the <code>hidden</code> attribute on the <code>nav</code>
-							element indicates the page list will be excluded from rendering in the content flow when the
-							document is rendered in the spine.</p>
+			<section id="sec-nav-doc-use-spine" class="informative">
+				<h3>Using in the Spine</h3>
 
-						<pre>&lt;nav
+				<p>Although it is possible to reuse the EPUB Navigation Document in the <a>spine</a>, it is often the
+					case that not all of the navigation structures, or branches within them, are needed. <a>EPUB
+						Creators</a> will often want to hide the <a href="sec-nav-pagelist">page list</a> and <a
+						href="sec-nav-landmarks">landmarks</a> navigation elements, or trim the branches of the table of
+					contents for books that have many levels of subsections.</p>
+
+				<p>While the <a href="https://www.w3.org/TR/CSS2/visuren.html#propdef-display"><code>display</code>
+						property</a> [[CSSSnapshot]] controls the visual rendering of EPUB Navigation Documents in
+					Reading Systems with <a>Viewports</a>, not all Reading Systems provide such an interface. To control
```

Comment:
I am not sure that the 'interface' is the usual terminology for CSS implementation (I looked at, for example, the [CSS21 Conformance part](https://www.w3.org/TR/CSS21/conform.html#q3.0). Maybe:

```suggestion
					Reading Systems with <a>Viewports</a>, not all Reading Systems implement it. To control
```

Reply:
Ah! Yes, that is much clearer, thanks!